kvbc/src/st_reconfiguration_sm.cpp

```diff
@@ -48,6 +47,13 @@ void StReconfigurationHandler::stCallBack(uint64_t current_cp_num) {
                                                             current_cp_num);
   handlerStoredCommand<concord::messages::AddRemoveWithWedgeCommand>(
       std::string{kvbc::keyTypes::reconfiguration_add_remove, 0x1}, current_cp_num);
+  handlerStoredCommand<concord::messages::PruneRequest>(std::string{kvbc::keyTypes::reconfiguration_pruning_key, 0x1},
+                                                        current_cp_num);
+}
+
+void StReconfigurationHandler::pruneOnStartup() {
```

Comment:
I was wondering who calls this function. Could you please elaborate?

Reply:
The replica calls this method once it is starting. 
This is for the case where the replica is restarting during the pruning

Reply:
Yep :)

Reply:
fixed